controls/SV-257844.rb

```diff
@@ -11,14 +11,25 @@
   desc 'fix', 'Migrate the "/tmp" path onto a separate file system.'
   impact 0.5
   ref 'DPMS Target Red Hat Enterprise Linux 9'
-  tag check_id: 'C-61585r925517_chk'
   tag severity: 'medium'
+  tag gtitle: 'SRG-OS-000480-GPOS-00227'
   tag gid: 'V-257844'
   tag rid: 'SV-257844r925519_rule'
   tag stig_id: 'RHEL-09-231015'
-  tag gtitle: 'SRG-OS-000480-GPOS-00227'
   tag fix_id: 'F-61509r925518_fix'
-  tag 'documentable'
   tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
+  tag 'host'
+
+  only_if('This control is Not Applicable to containers', impact: 0.0) {
+    !virtualization.system.eql?('docker')
+  }
+
+  describe mount('/tmp') do
+    it { should be_mounted }
+  end
+
+  describe etc_fstab.where { mount_point == '/tmp' } do
+    it { should exist }
```

Comment:
https://docs.chef.io/inspec/resources/etc_fstab/#check-if-a-partition-is-mounted-at-a-point

seemed like it ought to be be_configured not just exist

this comment applies to all the controls involving checking if a directory has its own partition, ex. so the home dir one too